src/ui/carousel/draggable-slide/matrix-service.ts

```diff
@@ -0,0 +1,103 @@
+import { Matrix } from './matrix';
+import { DefaultMap } from '../../../map/default-map';
+import { Raf } from '../../..';
+import { Vector } from '../../../mathf/vector';
+
+/**
+ * This service is used to coordinate modifications to a transform coming
+ * from multiple services. This allows us to stack transform-effecting UI
```

Comment:
Do we have any existing examples where two different pieces of code are modifying a single element's transforms?

Reply:
Yes. But not that I can post in the public repo.